### PR TITLE
Small ophys updates

### LINKED ISSUE #263: Small ophys items
- [x] make patch_cords Optional
- [x] Add camera to ophys session
- [x] Camera description/purpose
- [ ] Microscope class in device (what all does this need beyond base device class?)
- [x] magnification in FOV

### PR DESCRIPTION
**Pull request by saskiad**
closes #263

### CODE REVIEW

**Comment by saskiad** [comment]
camera assembly name already being used to specify the purpose - eg. Eye Camera vs Body Camera. BUT, I think it would be helpful to have a specific field with an ENUM that can be better used to identify cameras

**Comment by saskiad** [comment]
Until I have other field to add for microscope, I'm not adding it as a specific class and it will need to be an additional device. If there are other fields, then we can add it.

File: src/aind_data_schema/device.py [hunk 1]
```diff
@@ -240,12 +240,21 @@ class Filter(Device):
         None, title="Description", description="More details about filter properties and where/how it is being used"
     )
 
+class CameraTarget(Enum):
```
Comment (dyf) [inline src/aind_data_schema/device.py:243]: I recently polled for behavior videos, if you want to update the Enum:
image

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/aind_data_schema/device.py (hunk 1 above)
Comment (dyf) [reply src/aind_data_schema/device.py:243]: I think the only additions would be side vs body and tongue.

Comment (saskiad) [reply src/aind_data_schema/device.py:243]: Thanks. This allows confirms the need to have this field as relying on names won't yield useful searches!

**Review by dyf — APPROVED** [review]

**Pull request merged**